server/src/main/java/org/apache/cloudstack/consoleproxy/ConsoleAccessManagerImpl.java

```diff
@@ -151,11 +156,16 @@ public boolean isSessionAllowed(String sessionUuid) {
 
```

Comment:
@GutoVeronezi the problem listed previously for one mgmt server comes from the method `isSessionAllowed` that does not query the database for the session UUID and always returns false, therefore the `ConsoleAccessAuthenticationAnswer` is false

Reply:
@nvazquez, thanks. I missed a line while porting the changes to the PR. I fixed it with commit dec6e3fb461763605ca6be34013d7e2c2438827a and was doing some more testing before pinging you.